src/Extractor.php

```diff
@@ -69,6 +69,7 @@ public function run(Config $config, ?int $limit = null) : void
                         $report['displayOptions']['statGranularity'] ?? null
                     );
                 }
+                $primary = ($report['resource'] === 'queries') ? 'query' : 'id';
```

Comment:
Mas tu teraz 2x rovnaku podmienku, mozno by si to mohol dat do funkcie napr. `getPrimaryKey($resource)`.
Inak si myslim, ze je to OK.